config/plugins/mdx-plugin/index.js

```diff
@@ -0,0 +1,20 @@
+import { extname } from 'path';
```

Comment:
Do we want to ship our rollup mdx plugin (because right now the mdx loader is not available as npm published)? We already publish `/mdx-loader` for webpack. Perhaps we could ticket out converting that to `/mdx-loader/{webpack,rollup}` and publishing both from the same directory so folks can create Spectacle decks with either...

Reply:
That sounds like a great idea, the rollup plugin needs a little cleaning up (and sourcemaps need to be configured) before it's available for release but that's definitely something we should work on.